src/current/_includes/common/hybrid-multi-cloud.md

```diff
@@ -0,0 +1,66 @@
+{{site.data.alerts.end}}
+
+## Configure a hybrid or multi-cloud deployment
+
+To set up a hybrid or multi-cloud deployment:
+
+- **Each node must be able to communicate with and route traffic to each of the cluster's other nodes** using the hostnames or IP addresses referenced by the `--join` flag of the [`cockroach start`]({% link {{ page.version.version }}/cockroach-start.md %}) command. A node's network is configured at the level of the node's operating system. Nodes could be connected by peering cloud provider virtual private cloud (VPC) networks together, using a VPN, encrypted network tunnels, by a combination of routing and port forwarding, or using another mechanism.
+
+    CockroachDB could route SQL client requests to any cluster node, regardless of where the client connection originates.
+- **There must be no overlapping IP address ranges** across the cluster. It is an error for multiple nodes to have the same IP address or to resolve to the same hostname.
+
+    DNS name resolution is particularly complex in a Kubernetes deployment. On GKE, we recommend that you replace `kube-dns` with Core DNS. Refer to [Deploy CockroachDB on GKE](https://github.com/mbookham7/crdb-multi-cloud-k8s/blob/master/markdown/5-deploy-cockroach.md) for details.
+- **In a multi-region deployment, we recommend that you use manifests**. Refer to [Deploy multi-cloud CockroachDB on GKE](https://github.com/mbookham7/crdb-multi-cloud-k8s/blob/master/markdown/5-deploy-cockroach.md) for details.
+- **Each node deployed in the same environment must share locality** to ensure that the cluster's replicas are spread across deployment environments and to prevent single points of failure or hot spots. To specify a node's locality, use the `--locality` flag, which accepts an arbitrary set of key-value pairs that describe the location of the node. For example, if nodes are deployed in both Azure and Digital Ocean, you could set each node's locality to either `--locality data-center=azure` or `--locality data-center=digital-ocean`.
+
+    If nodes are deployed in multiple regions within the same cloud provider, specify the region as an additional locality, to ensure that replicas are spread across regions in each cloud provider's infrastructure.
+
+## Simulate an outage
+
+This section shows some ways to simulate an outage and validate disaster recovery procedures in a hybrid or multi-cloud deployment.
+
+- **To simulate a single-node outage**, you could down the host or VM where the node is running, or disable its network interface.
+- **To simulate a single-region cloud-provider outage**, you could shut down all hosts or VMs in that region on that cloud provider, or disable their network interfaces.
+- **To simulate a cloud-provider-wide outage**, you could shut down all hosts or VMs on that cloud provider, disable their network interfaces, or disconnect that cloud provider's network from your other nodes' networks.
```

Comment:
```suggestion
- **To simulate a cloud-provider-wide outage**, you could forcibly shut down the `cockroach` process all hosts or VMs running in that cloud provider, you could disable their network interfaces, adjust routing or firewall rules, or make other networking adjustments to prevent nodes in that cloud provider from connecting to the cluster.
```